completion/README.md

```diff
@@ -6,6 +6,19 @@ Make sure you have bash completion installed (with `brew install bash-completion
 
 `brew install git-machete` automatically installs bash completion files for `git machete`.
 
+If the bash completion doesn't work:
+1. Install git bash completions with `brew install git bash-completion`
+2. Add the below code to `~/.bash_profile` and reload the shell
+```shell script
+[ -f /usr/local/etc/bash_completion ] && . /usr/local/etc/bash_completion || {
+    # if not found in /usr/local/etc, try the brew --prefix location
+    [ -f "$(brew --prefix)/etc/bash_completion.d/git-completion.bash" ] && \
+    . $(brew --prefix)/etc/bash_completion.d/git-completion.bash
```

Comment:
Nit: indent the continuation lines like this one by one level, so that it's clear it's a continuation

Reply:
Or even better, extra indent + move `&&` from the previous line to the start of this line

Reply:
@PawelLipski better now: 
<img width="610" alt="image" src="https://user-images.githubusercontent.com/86779115/171214335-003178f1-1fd8-485d-ad3f-6f47b2159e1a.png">

Reply:
Huh but this isn't a valid shell script now! Try executing that